app/views/bulkrax/importers/_csv_spot_fields.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class='csv_spot_fields'>
+
+  <%= fi.input :visibility,
+    collection: [
+      ['Public', 'open'],
+      ['Private', 'restricted']
+    ],
+    selected: importer.parser_fields['visibility'] || 'open',
+    input_html: { class: 'form-control' }
+  %>
+
+  <% rights_statements = Hyrax.config.rights_statement_service_class.new %>
+  <%= fi.input :rights_statement,
+        collection: rights_statements.select_active_options,
+        selected: importer.parser_fields['rights_statement'],
+        include_blank: true,
+        item_helper: rights_statements.method(:include_current_value),
+        input_html: { class: 'form-control' },
+        required: false
+        %>
+  <%= fi.input :override_rights_statement, as: :boolean, hint: 'If checked, always use the selected rights statment. If unchecked, use rights or rights_statement from the record and only use the provided value if dc:rights is blank.', input_html: { checked: (importer.parser_fields['override_rights_statement'] == "1") } %>
+
+  <h4>Add CSV File to Import:</h4>
```

Comment:
if we establish a bucket layout for all imports (requiring a "metadata.csv" file and a "files" subdirectory), I wonder if it might be easier for the end user to replace this text field with a dropdown of top level keys within the ingest bucket (using `s3_client.list_objects(bucket: ingest_bucket, delimiter: '/')`)